cypress/support/PersistentLogs.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+
+var testCaseRegistry = [];
+var indexRegistry = 0;
+var logEntries = '';
+
+// Override the original console.log() function
+var originalConsolLog = console.log;
+
+// This becomes the effective console.log() function.
+console.log = function (message) {
```

Comment:
Does cypress offer any built in features that can save stdout as a log file? 

It's generally recommended to avoid overriding browser defaults and instead use a custom function so people expect it will behave differently and are not misled. I though you were already using a custom `helpers.conLog` everywhere? Can we just extend that one do write add to the file instead?

I guess in this case it ensures people can't use the regular log and forget to use the helper and mistakenly avoid writing the file.

Reply:
Well Cypress does have plans of including this feature in their code, but it is not ready yet. I discovered that just yesterday while researching another issue. **So when theirs is released we can just drop this file and everything will continue to work.** Which is one good reason not to change our helper method.